test/plugin/scenarios/agent-so11y-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,182 @@
+      - {key: db.statement, value: DROP table test_007}
+      skipAnalysis: 'false'
+    - operationName: /agent-so11y-scenario/case/ignore.html
+      parentSpanId: 0
+      spanId: 4
+      spanLayer: Http
+      startTime: gt 0
+      endTime: gt 0
+      componentId: 66
+      isError: false
+      spanType: Exit
+      peer: localhost:8080
+      tags:
+        - {key: http.method, value: GET}
+        - {key: url, value: 'http://localhost:8080/agent-so11y-scenario/case/ignore.html'}
+        - {key: http.status_code, value: '200'}
+      skipAnalysis: 'false'
+    - operationName: /agent-so11y-scenario/case/propagated
+      parentSpanId: 0
+      spanId: 5
+      spanLayer: Http
+      startTime: gt 0
+      endTime: gt 0
+      componentId: 66
+      isError: false
+      spanType: Exit
+      peer: localhost:8080
+      tags:
+        - {key: http.method, value: GET}
+        - {key: url, value: 'http://localhost:8080/agent-so11y-scenario/case/propagated'}
+        - {key: http.status_code, value: '200'}
+      skipAnalysis: 'false'
+    - operationName: GET:/agent-so11y-scenario/case/agent-so11y-scenario
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: gt 0
+      endTime: gt 0
+      componentId: 1
+      isError: false
+      spanType: Entry
+      peer: ''
+      tags:
+      - {key: url, value: 'http://localhost:8080/agent-so11y-scenario/case/agent-so11y-scenario'}
+      - {key: http.method, value: GET}
+      - {key: http.status_code, value: '200'}
+      skipAnalysis: 'false'
+  - segmentId: not null
+    spans:
+    - operationName: GET:/agent-so11y-scenario/case/propagated
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: gt 0
+      endTime: gt 0
+      componentId: 1
+      isError: false
+      spanType: Entry
+      peer: ''
+      tags:
+      - {key: url, value: 'http://localhost:8080/agent-so11y-scenario/case/propagated'}
+      - {key: http.method, value: GET}
+      - {key: http.status_code, value: '200'}
+      skipAnalysis: 'false'
+      refs:
+      - {parentEndpoint: 'GET:/agent-so11y-scenario/case/agent-so11y-scenario', networkAddress: 'localhost:8080',
+        refType: CrossProcess, parentSpanId: 5, parentTraceSegmentId: not null,
+        parentServiceInstance: not null, parentService: agent-so11y-scenario,
+        traceId: not null}
+
+meterItems:
+- serviceName: agent-so11y-scenario
+  meterSize: ge 9
+  meters:
+  - meterId:
+      name: created_tracing_context_counter
+      tags:
+      - {name: created_by, value: propagated}
+    singleValue: 1.0
+  - meterId:
+      name: created_tracing_context_counter
+      tags:
+      - {name: created_by, value: sampler}
+    singleValue: 1.0
+  - meterId:
+      name: finished_tracing_context_counter
+      tags: []
+    singleValue: 2.0
+  - meterId:
+      name: created_ignored_context_counter
+      tags:
+      - {name: created_by, value: propagated}
+    singleValue: 1.0
```

Comment:
Is counter value 1 stable?

Reply:
Yes, this propagated context is created by `new URL("http://localhost:8080/agent-so11y-scenario/case/propagated").getContent();`